src/Commands/SetLifter.cpp

```diff
@@ -0,0 +1,40 @@
+#include "SetLifter.h"
+
+SetLifter::SetLifter(Shooter::LifterValue)
+{
+mLifterValue = Shooter::LifterValue;
```

Comment:
`Shooter::LifterValue` is a type. You need a parameter name. Above I create a parameter name `lifterValue`. This line now becomes:

`mLifterValue = lifterValue;`

Better would be to use an initializer list:

```
SetLifter::SetLifter(Shooter::LifterValue lifterValue)
: mLifterValue(lifterValue)
{
}
```